### PR TITLE
Create Shoutwiki.com

### PR DESCRIPTION
**Pull request by davidabian**

### CODE REVIEW

File: src/chrome/content/rules/Shoutwiki.com.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<ruleset name="ShoutWiki.com" platform="mixedcontent">
+
+	<target host="shoutwiki.com" />
+	<target host="*.shoutwiki.com" />
+
+	<test url="http://es.shoutwiki.com/" />
+	<test url="http://www.shoutwiki.com/" />
+	<test url="http://blog.shoutwiki.com/" />
+	<test url="http://bugzilla.shoutwiki.com/" />
+	<test url="http://fi.starwars.shoutwiki.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Shoutwiki.com.xml]: Your rule will break all 4th level domain, since they serve no valid cert.

**Comment by J0WI** [comment]
You should whitelist working ones instead of using weird and buggy exclusions.

**Comment by jeremyn** [comment]
@davidabian Are you still interested in working on this one?

**Comment by galeksandrp** [comment]
#4378

**Comment by bardiharborow** [comment]
In 2017, the only subdomain without a certificate mismatch is `blog`. This is probably closable.

**Comment by J0WI** [comment]
@davidabian would be great if you could have a further look at this PR.

**Comment by jeremyn** [comment]
I'm closing this due to no response from @davidabian and to an apparent lack of valid certificates.

**Pull request closed without merging**